examples/matrix/cluster-and-git.yaml

```diff
@@ -0,0 +1,34 @@
+# This example demonstrates the combining of the git generator with a clusgter generator
+# The expected output would be an application per git directory and a clsuter (applicztions_count = git directory * clusters)
+#
+#
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: cluster-git
+spec:
+  generators:
+  - matrix:
+    generators:
```

Comment:
Indentation is off in these examples, it looks like `generators:`, and all its children, should be shifted to the right by 2.